server/dockrun.sh

```diff
@@ -17,7 +17,7 @@ elif (($# == 3)); then
 	SRCDIR=$1
 	HOSTPORT=$2
 	IMAGE_NAME=$3
-	CONTAINER_ID=pp
+	CONTAINER_ID=${4:-pp} #defaults to 'pp' if not provided
```

Comment:
Since the 4th argument is not being used, can keep using the hardcoded `pp` value as $CONTAINER_ID. Also, `server/dockrun.sh` should be deleted, it was moved to `container/run.sh` and the comments in that file should reflect the file rename.